common-tasks/working-with-office/en.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Working with office"
++++
+# Working with office
+
+Solus comes pre-installed with LibreOffice, an office suite that is capable of replacing Microsoft Office for the most common tasks.
```

Comment:
> Microsoft Office

Should be Microsoft<sup>TM</sup> Office (`Microsoft<sup>TM</sup> Office`). According to their [trademarks](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/en-us.aspx), they don't have a trademark itself for Office, just versions (like 2012) or software in it, ex. Microsoft Word.